client/src/pages/Post/PostPage.js

```diff
@@ -72,7 +68,16 @@ class PostPage extends React.Component {
             <div>
                 <div className={classes.post}>
                     <Post
-                        handleSave={this.save}
+                        handleSave={
+                            e => {
+                                e.preventDefault();
+                                axios.put(`users/board/${this.state.board}`, {
+                                    object_id: `${this.state.id}`
```

Comment:
I think it should be `_id` instead of `object_id`

Reply:
Where does the user token go in this request? It seems like it isn't authenticated, but the server expects it to be

Reply:
This logic shouldn't be in the render method btw, move it to its own function

Reply:
in src/services/user addTokenHeaders adds token to axios for all requests. If you could move the api request to src/services/board as well.

Reply:
I see. How about on services you export an instance of axios with the token header set. Then for any authenticated request, you would import axios from services to use that instance instance of "axios"